src/sections/Community/Handbook/repo-data.js

```diff
@@ -398,20 +398,56 @@ export const repo_data = [
   },
 
   {
-    category: "Installers Repos",
+    category: "Maintainers",
     subdata: [
       {
-        project: "Scoop Bucket",
-        language: "",
-        description: "Windows-compatible package management for `mesheryctl`",
-        repository: "https://github.com/layer5io/scoop-bucket",
+        project: "Service Mesh Pattern",
+        image: patterns,
+        site: "https://service-mesh-patterns.github.io/service-mesh-patterns/",
+        maintainers: "Nikhil Ladha",
+        repository: "https://github.com/service-mesh-patterns/service-mesh-patterns",
+      },
+      {
+        project: "Meshery.io",
+        image: meshery,
+        site: "meshery.io",
+        maintainers: "Aditya Chatterjee",
+        repository: "https://github.com/meshery/meshery.io",
       },
       {
-        project: "Homebrew Tap",
-        image: brew,
-        language: "Ruby",
-        description: "MacOS-compatible package management for `mesheryctl`",
-        repository: "https://github.com/layer5io/homebrew-tap",
+        project: "Discuss Layer5.io",
+        image: five,
+        site: "meshery.io",
+        maintainers: "Leonard Nwobodo",
+        repository: "http://discuss.layer5.io/",
+      },
+      {
+        project: "Layer5.io",
+        image: five,
+        site: "http://layer5.io/",
+        maintainers: "Debopriya Bhattacharjee,Gaurav Chadha, Nikhil Ladha Aditya Chatterjee",
+        repository: "https://github.com/layer5io/layer5",
+      },
+      {
+        project: "Service Mesh Pattern-Spec",
```

Comment:
```suggestion
        project: "Service Mesh Performance",
```
SMP = Service Mesh Performance. Make sure you use this in naming the images as well.

Reply:
ok I didnt knew this thanks